Moified the organization of the code and removed special keys

IonutCicio opened the pull request.

IonutCicio committed:
Moified the organization of the code, removed special keys and added two buttons

IonutCicio committed:
Add files via upload

wbigger commented:
Purpose seems good but the names of the files have something strange. Each file should contain a "concept", with both data (variables) and corresponding function for that concept. If a file contains just some function that are utilities for another concept (i.e. class), consider to name it "SomethingUtils" or similar. However, try to keep low the number of these file, preferring a conceptual decomposition of your code in sub-concept or modules.

wbigger commented:
Try to search "why to avoid utility classes" on google to have an idea.

IonutCicio committed:
Delete Wireframe_layout_keyboard.docx

IonutCicio committed:
Delete Wireframe_layout_settings_menu.docx

IonutCicio committed:
Delete Wirefram_layout_exercise_page.docx

IonutCicio committed:
Changed the documentation

-Added "Current mode" in the documentation.
-Added "Wireframe_layout_main_meu".
-Changed "Wireframe_layout_settings_menu" (behavior and design)
-Changed "Wireframe_layout_exercise_page" with the middle bar, and the counters.
-Added more ideas to develop in "Wireframe_layout_progress_menu".
-I'm still thinking at progress' menu wireframe layout.

IonutCicio committed:
Delete Personas.docx

IonutCicio committed:
Delete User stories.docx

IonutCicio committed:
Delete Brief.docx

IonutCicio committed:
Delete Brief.docx

IonutCicio committed:
Delete Personas.docx

IonutCicio committed:
Delete User stories.docx

IonutCicio committed:
Delete Wireframe_layout_exercise_page.docx

IonutCicio committed:
Delete Wireframe_layout_keyboard.docx

IonutCicio committed:
Delete Wireframe_layout_main_menu.docx

IonutCicio committed:
Delete Wireframe_layout_progress_page.docx

IonutCicio committed:
Delete Wireframe_layout_settings_menu.docx

IonutCicio committed:
Changes in Documentation and added some features:

-DOCUMENTATION:
 -changed the design of "Wireframe_layout_main_menu.docx".
 -changed the design of "Wireframe_layout_settings_menu.docx".
 -changed the design and the behavior of "Wireframe_layout_progress_menu.docx", but I still have to change the design by adding 
 "currentMode" and "currentUser".
 -created "Settings.txt", but it still doesn't work. The code to read it will be add soon.
-FEATURES:
 -added ALL indicators, working very well, but they need some adjustments.
 -added writable text, but still not completed the "writing text" part.
 -changed the way to color "Boxes", "Buttons" and "Keys".
 -transformed "currentMode" in to a "Box".
 -created "currentUser" "Box".
 -that's preatty much all.

IonutCicio committed:
Create .GITIGNORE

IonutCicio committed:
Delete .GITIGNORE

IonutCicio committed:
Create .gitignore

IonutCicio committed:
Add files via upload

IonutCicio committed:
Hard mode working

IonutCicio committed:
Changed some documentation
